lib/mdns_lite/options.ex

```diff
@@ -99,7 +99,11 @@ defmodule MdnsLite.Options do
   @doc false
   @spec from_application_env() :: t()
   def from_application_env() do
-    hosts = Application.get_env(:mdns_lite, :host, @default_host_name_list)
+    # This used to be called :host, but now it's :hosts
+    hosts =
+      Application.get_env(:mdns_lite, :hosts) || Application.get_env(:mdns_lite, :host) ||
```

Comment:
Do you think we need to warn if they are using the `:host` key?

Reply:
Also, how did this work before?! If someone is still using `config :mdns_lite, host: :hostname` it seems like this will be broken because there is a hard expectation on it being a list. I think we might need to normalize the `:host` key here if we want to continue supporting it?

Or is the idea to just let it fail if it is not a list?

Reply:
It was a list when it was `:host` as well. That's why I changed it since it bothered me a lot.

There are tons of configs out there with `:host`, so I was initially leaning away from giving a warning, but you're making me think that this is so confusing and unexpected that I better give the warning and normalize the key.

Reply:
O! So we've totally just had bad outdated docs out there for a while. Dang 🤦 

I think it's been long enough that we're prob okay here

Reply:
I updated it anyway. I'm quite sure that you're not alone and it's not something I really want to spend time debugging.